packages/kbn-test/src/functional_test_runner/lib/docker_servers/README.md

```diff
@@ -0,0 +1,85 @@
+# Functional Test Runner - Docker Servers
+
+In order to make it simpler to run some services while the functional tests are running, we've added the ability to execute docker containers while the tests execute for the purpose of exposing services to the tests. These containers are expected to expose an application via a single HTTP port and live for the life of the tests. If the application exits for any reason before the tests complete the tests will abort.
+
+To configure docker servers in your FTR config add the `dockerServers` key to your config like so:
+
+```ts
+// import this helper to get TypeScript support for this section of the config
+import { defineDockerServersConfig } from '@kbn/test';
+
+export default function () {
```

Comment:
nit:
```suggestion
export default async function () {
```

Reply:
The `await` is within the `waitFor()` function though ;)